CurrencyConverter/CurrencyConverter/ViewController.swift

```diff
@@ -6,34 +6,101 @@
 //  Copyright © 2019 Lambda School. All rights reserved.
 //
 
+
 import UIKit
 
-class ViewController: UIViewController {
-    
-    // MARK: - Outlets/Properties
+class CurrencyConvertorViewController: UIViewController {
     
-    @IBOutlet weak var fromCurrencyTextField: UITextField!
-    @IBOutlet weak var toCurrencyTextField: UITextField!
+    @IBOutlet weak var pesoCurrencyLabel: UILabel!
+    @IBOutlet weak var pesoCurrencyTextField: UITextField!
+    @IBOutlet weak var usCurrencyLabel: UILabel!
+    @IBOutlet weak var usCurrencyTextField: UITextField!
+    @IBOutlet weak var calculateButton: UIButton!
     
-    @IBOutlet weak var toCurrencyLabel: UILabel!
     
-    @IBOutlet weak var cadButton: UIButton!
-    @IBOutlet weak var mxnButton: UIButton!
-
-    // MARK: - Actions
-    
-    @IBAction func convertButtonTapped(_ sender: UIButton) {
+    override func viewDidLoad() {
+        super.viewDidLoad()
         
-    }
-    
-    @IBAction func cadButtonTapped(_ sender: UIButton) {
+        usCurrencyLabel.text = "Currency (US)"
         
-    }
-    
-    @IBAction func mxnButtonTapped(_ sender: UIButton) {
+        if segmentedControl.selectedSegmentIndex == 0 {
+            pesoCurrencyLabel.text = "Currency (PESO)"
+        } else {
+            pesoCurrencyLabel.text = "Currency (CAD)"
+        }
         
-    }
-    
-    // MARK: - Helper Methods
+        
+        
+        class CurrencyConvertorViewController: UIViewController {
```

Comment:
You're making a new class of CurrencyConvertorViewController within a class of CurrencyViewController. Your IBOutlets and actions below don't function properly until the new class is deleted or placed outside of the first one. In the storyboard it looks like the connections to the outlets are broken and I think that's the problem.